models/consultation.py

```diff
@@ -1,7 +1,29 @@
+class ConsultationTemplate:
+    @staticmethod
+    def convert(consultation):
+        return f"👨‍🏫 Учитель {consultation.teacher} \nКабинет: {consultation.room}\n✉ Почта: {consultation.email}" \
+               f"\nОтрасль: {consultation.department} " \
+               f"\n🗓 День недели: {consultation.weekday}\n⏰ Время: {consultation.time}\n"
+
+
+class TeacherSearchingTemplate:
+    @staticmethod
+    def convert(consultation):
+        return f"{consultation.teacher}"
+
+
 class Consultation:
-    def __init__(self, teacher, room, email, department, times):
+    def __init__(self, teacher, room, email, department, times, weekday, time):
```

Comment:
Better to define *time* array for this object. Like: 1 teacher have 2-5 times of consultations.